openslides_backend/presenter/search_users_by_name_email.py

```diff
@@ -0,0 +1,143 @@
+from typing import Any, Dict, List, Tuple
+
+import fastjsonschema
+
+from ..action.actions.user.user_scope_permission_check_mixin import UserScope
+from ..permissions.management_levels import (
+    CommitteeManagementLevel,
+    OrganizationManagementLevel,
+)
+from ..permissions.permission_helper import (
+    has_committee_management_level,
+    has_organization_management_level,
+    has_perm,
+)
+from ..permissions.permissions import Permissions
+from ..shared.exceptions import MissingPermission, PresenterException
+from ..shared.filters import FilterOperator, Or
+from ..shared.patterns import Collection, FullQualifiedId
+from ..shared.schema import schema_version
+from .base import BasePresenter
+from .presenter import register_presenter
+
+search_users_by_name_email_schema = fastjsonschema.compile(
+    {
+        "$schema": schema_version,
+        "type": "object",
+        "title": "search_users_by_name_email",
+        "description": "get lists of id, first-, last-name and email for tuples of exact (username, emails)-tuples.",
+        "properties": {
+            "permission_scope": {
```

Comment:
Will this ever be called from a commitee or organization scope? Isn't the meeting scope sufficient since the feature is meant for meeting admins?

Reply:
Then, you could just drop the whole `permission_scope` section and just provide a `meeting_id` and check below whether the user has permissions for this meeting, the meeting's committee oder the whole organization.